docs/user/openstack/deploy_sriov_workers.md

```diff
@@ -0,0 +1,356 @@
+
+## Preparing your OpenShift Cluster to use SR-IOV Networks
+
+Once your OpenShift control plane is running, you will need to install the SR-IOV Network Operator. To do so, you will need access to an account on your OpenShift cluster with `cluster-admin` privileges. Once you are logged into that account, follow [these steps](sriov-operator) to install the Operator.
+
+Next, you will need to create a machine config that enables the SR-IOV operator to get the UUID of networks from OpenStack. This machine config will allow the OpenStack metadata service to be available as a mountable drive.
+
+```yaml
+kind: MachineConfig
+apiVersion: machineconfiguration.openshift.io/v1
+metadata:
+  name: 20-mount-config
+  labels:
+    machineconfiguration.openshift.io/role: worker
+spec:
+  osImageURL: ''
+  config:
+    ignition:
+      version: 2.2.0
+    systemd:
+      units:
+        - name: create-mountpoint-var-config.service
+          enabled: true
+          contents: |
+            [Unit]
+            Description=Create mountpoint /var/config
+            Before=kubelet.service
+
+            [Service]
+            ExecStart=/bin/mkdir -p /var/config
+
+            [Install]
+            WantedBy=var-config.mount
+
+        - name: var-config.mount
+          enabled: true
+          contents: |
+            [Unit]
+            Before=local-fs.target
+            [Mount]
+            Where=/var/config
+            What=/dev/disk/by-label/config-2
+            [Install]
+            WantedBy=local-fs.target
+```
+
+You can apply a machine config to your machine pool that enables the No-IOMMU feature for the Red Hat OpenStack Platform (RHOSP) virtual function I/O (VFIO) driver. The RHOSP vfio-pci driver requires this feature.
+
+```yaml
+kind: MachineConfig
+apiVersion: machineconfiguration.openshift.io/v1
+metadata:
+  name: 99-vfio-noiommu
+  labels:
+    machineconfiguration.openshift.io/role: worker
+spec:
+  osImageURL: ''
+  config:
+    ignition:
+      version: 2.2.0
+    storage:
+      files:
+      - filesystem: root
+        path: "/etc/modprobe.d/vfio-noiommu.conf"
+        contents:
+          source: data:text/plain;charset=utf-8;base64,b3B0aW9ucyB2ZmlvIGVuYWJsZV91bnNhZmVfbm9pb21tdV9tb2RlPTEK
+          verification: {}
+        mode: 0644
+```
+
+If you need to configure your deployment for real-time running or low latency, see the [performance addon operator documentation](perfocmance-addon-operator).
+
+## Creating SR-IOV Networks for Compute Machines
+
+The SR-IOV networks that are attached to compute nodes need to be created before creating the nodes. Please reference the example of how to create radio and uplink provider networks in OpenStack.
```

Comment:
use workers not "compute nodes" or "nodes" to be  consistent.

Reply:
I should clarify that. I use radio and uplink as examples throughout the doc just to keep the examples consistent and easy to follow.